lambdas/db-migration/src/migrations/20201014105102_create_files_table.ts

```diff
@@ -28,8 +25,6 @@ export const up = async (knex: Knex): Promise<void> =>
     table
       .text('checksum_value')
       .comment('File checksum');
-    table
```

Comment:
Based on FileUtils.js, it looks like `name` might be another duplicate of `filename`/`file_name`. Do you mind taking a looking at that?

Reply:
Oh, that's true. Good catch! Yes, `name` seems to also be a duplicate, so I've since removed it [`93f0b01` (#2037)](https://github.com/nasa/cumulus/pull/2037/commits/93f0b01ce29fd078dbb1c7a05dac7dcd02e8da18)